### ISSUE TITLE
[DUPE of 13736?] Storefinder button with meaningless SR narration

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#6767]
When focusing on **City** buttons, Screen Reader's narration should be meaningful, currently, it only vocalizes "Get Directions", event though a mouse user can see which city is being focused, a screen reader user will not know to which city the button belongs to. Same case when using the VO Rotor, there will be a meaningless list of "Get Directions" links.

This needs an aria-label including the name of the store and preventing the user that it points to an external link and opens a new window.

**Video demo with SR narration:** https://youtu.be/SuGzG10zcT0

![Screen Shot 2020-02-28 at 9 11 47 PM](https://user-images.githubusercontent.com/15113219/75726913-9bbc5800-5cb1-11ea-80d0-f8a78eff1d9e.png)

**Comment by saad-mo** [comment]
Duplicate of #13736. The same component is used in both cases (`cx-store-finder-list-item`) and fixing #13736 will also fix this issue.
